table/src/main/java/tech/ydb/table/description/TableTtl.java

```diff
@@ -58,4 +98,48 @@ public static TtlMode forCase(int value) {
             throw new IllegalArgumentException("No TTL mode defined for specified value");
         }
     }
+
+    public enum TtlUnit {
+        UNSPECIFIED,
+        SECONDS,
+        MILLISECONDS,
+        MICROSECONDS,
+        NANOSECONDS;
+    }
+
+    /**
+     * Construct an empty TTL configuration
+     *
+     * @return instance of TTL configuration
+     */
+    public static TableTtl notSet() {
+        return NOT_SET;
+    }
+
+    /**
+     * The row will be considered as expired at the moment of time, when the value stored in <i>columnName</i> is less
+     * than or equal to the current time (in epoch time format), and <i>expireAfterSeconds</i> has passed since that
+     * moment; i.e. the expiration threshold is the value of <i>columnName</i>plus <i>expireAfterSeconds</i>.
+     *
+     * @param columnName name of column with type Date, Datetime or Timestamp
+     * @param expireAfterSeconds number of seconds to add to the time in the column
+     * @return instance of TTL configuration
+     */
+    public static TableTtl dateTimeColumn(@Nonnull String columnName, int expireAfterSeconds) {
+        return new TableTtl(TtlMode.DATE_TYPE_COLUMN, TtlUnit.UNSPECIFIED, columnName, expireAfterSeconds, null);
+    }
+
+    /**
+     * The row will be considered as expired at the moment of time, when the value stored in <i>columnName</i> is less
+     * than or equal to the current time (in epoch time format), and <i>expireAfterSeconds</i> has passed since that
+     * moment; i.e. the expiration threshold is the value of <i>columnName</i>plus <i>expireAfterSeconds</i>.
+     *
+     * @param columnName name of column with type UInt32, UInt64 or DyNumber
+     * @param unit time unit  of column
+     * @param expireAfterSeconds number of seconds to add to the time in the column
+     * @return instance of TTL configuration
+     */
+    public static TableTtl valueSinceUnixEpoch(@Nonnull String columnName, TtlUnit unit, int expireAfterSeconds) {
```

Comment:
Maybe it's just expireAfter, if we pass time unit?
Same for other places where timeUnit is used

Reply:
It is made specially to avoid a mistake you made. Clients can set the time unit for  the value column, but expireAfter is always counting in seconds, no matter what time unit is used for the value in the column